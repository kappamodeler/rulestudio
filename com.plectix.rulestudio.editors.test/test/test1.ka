# this is a test file

a(s1,s2)-> a(s1!1,s2!1) 

a(s1,s2,s3)->a(s1,s2) 

%init

   %init \
more

