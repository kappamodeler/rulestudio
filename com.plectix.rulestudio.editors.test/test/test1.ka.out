# Created Tue Dec 22 18:44:20 EST 2009
# start	end	level	message
Line 7, Char 1	79	85	Error	Invalid declaration, missing :
Line 9, Char 1	91	105	Error	Invalid declaration, missing :
